src/devicecontroller/DefaultDeviceController.ts

```diff
@@ -1170,6 +1170,10 @@ export default class DefaultDeviceController implements DeviceControllerBasedMed
     } else if (stream) {
       // @ts-ignore - create a fake track constraint using the stream id
       trackConstraints.streamId = stream.id;
+    } else if (DefaultDeviceController.isMediaDeviceInfo(device)) {
+      // trackConstraints.deviceId = device.deviceId;
+      // trackConstraints.groupId = device.groupId;
```

Comment:
?

Reply:
My bad. I commented it out to test unit test and forgot to revert it back